### DISCUSSION TITLE
DeepMD-kit | The problem of  key word in v.2.0.0

### DISCUSSION THREAD

**Discussion by walkjoker-c** [deepmodeling/deepmd-kit#820] [Q&A]
UserWarning: It seems that you are using a deepmd-kit input of version 1.x.x, which is deprecated. we have converted the input to >2.0.0 compatible, and output it to file input_v2_compat.json

![$G8D4W 2D%4P8S6IL6X9HHY](https://user-images.githubusercontent.com/74398479/124090785-beb67200-da87-11eb-9860-9461d997a10a.png)

So what is the keyword  in version v2.0.0

**Comment by AnguseZhang** [comment]
Please refer to devel branch's manual and description.
https://github.com/deepmodeling/deepmd-kit/blob/devel/doc/train-se-e2-a.md#train-a-deep-potential-model

**Comment by njzjz** [comment]
See also https://docs.deepmodeling.org/projects/deepmd/en/latest/train-input.html.
